src/components/Patient/PatientHome.tsx

```diff
@@ -137,7 +137,14 @@ export const PatientHome = (props: {
           </div>
         </div>
         <div className="lg:flex">
-          <div className="h-full lg:mr-7 lg:basis-5/6">
+          <div
+            className={cn(
+              "h-full",
+              Tab?.component === PatientNotesTab
```

Comment:
let's change the comparison here and below (line 158) to use tab.route instead, similar to 151.